release/release_tests.yaml

```diff
@@ -3322,6 +3322,8 @@
   group: core-daily-test
   working_dir: nightly_tests
 
+  stable: false
```

Comment:
can you add a todo/github issue for fixing, thankks

Reply:
Done